include/flamegpu/model/LayerDescription.h

```diff
@@ -219,19 +223,20 @@ class LayerDescription : public CLayerDescription {
      * @throw exception::InvalidHostFunc If the function has already been added to the layer
      * @note ONLY USED INTERNALLY AND BY PYTHON API - DO NOT CALL IN C++ BUILD
      */
-    void _addHostFunctionCallback(HostFunctionCallback *func_callback);
+    inline void addHostFunction(HostFunctionCallback *func_callback);
```

Comment:
Just checking this is still intentionally `HostFunctionCallback` in the c++ portion (This is the first occurance in the diff). 

Its fine it if is, just thought i'd highlight it.